_dev/src/ts/components/ComponentAbstract.ts

```diff
@@ -0,0 +1,13 @@
+export default abstract class ComponentAbstract {
+  public element: HTMLElement;
```

Comment:
```suggestion
  protected element: HTMLElement;
```

Reply:
If I change it to protected I can't test it and after compilation it changes nothing.

Reply:
you want to set a private variable and get if from getter right ? That's possibility but this can make the process slightly heavy

Reply:
I especially want to avoid having public class variables, it goes against the fundamental principles of encapsulation and modularity in object-oriented programming.